dhis-2/dhis-api/src/main/java/org/hisp/dhis/program/EnrollmentStore.java

```diff
@@ -44,18 +44,18 @@
     String ID = EnrollmentStore.class.getName();
 
     /**
-     * Count all enrollments by PI query params.
+     * Count all enrollments by enrollment query params.
      *
      * @param params EnrollmentQueryParams to use
-     * @return Count of matching PIs
+     * @return Count of matching Enrollments
```

Comment:
```suggestion
     * @return Count of matching enrollments
```

I would consistently use lowercase unless unless its at the start of a sentence 🙂